### ISSUE TITLE
iframe in slide

### ISSUE THREAD

**Issue by alexandercbooth** [FormidableLabs/spectacle#212]
First, I'm loving Spectacle. Many thanks to @kenwheeler and all the contributors -- it's fantastic. 

How would you recommend incorporating iframes into a slide (I am coming from using Reveal.js and with those presentations I would normally use iframes for D3 visualizations & asciinema movies)? 

Thanks so much!

**Comment by kenwheeler** [comment]
Thanks @alexandercbooth ! So spectacle is just react, meaning you can just drop an iframe in. That said, another plus is, you can run interactive content right there on the slide, using Victory for viz and Component Playground to fiddle with it. See an example here: reacteurope.surge.sh

**Comment by alexandercbooth** [comment]
Thanks @kenwheeler, I'll definitely checkout victory! And I love the idea of component playground (I always try to make presentations with as dynamic content as possible). 

I can get iframes to work that render an outside page, but when I try to get them to load a local page that I have created, say in a folder `test/index.html`, it only renders a smaller version of the presentation itself. I think it's just a silly error on my part with the path, but where should I place my  `test` folder in the spectacle-boilerplate directory and how should I call it in the frame itself?

At the moment, I have placed the folder in the base directory of `spectacle-boilerplate` and am calling it in my `presentation/index.js` like this:
```jsx
<Slide>
   <iframe src="../test.html"></iframe>
</Slide>
```

Thanks so much!
